Source/Evolve/WarpXEvolveEM.cpp

```diff
@@ -270,7 +270,7 @@ WarpX::EvolveEM (int numsteps)
 
     bool write_plot_file = plot_int > 0 && istep[0] > last_plot_file_step
         && (max_time_reached || istep[0] >= max_step);
-    bool write_openPMD = openpmd_int > 0 && (max_time_reached || istep[0] >= max_step);
+    bool write_openPMD = (openpmd_int > 0) && (max_time_reached || istep[0] >= max_step);
```

Comment:
I actually thought you could just add a `last_openPMD_step` variable that is missing here (similar to `last_plot_file_step`).